src/amber/controller/helpers/responders.cr

```diff
@@ -50,8 +50,12 @@ module Amber::Controller::Helpers
 
       private def select_type
         raise "You must define at least one response_type." if @available_responses.empty?
-        # NOTE: If only one response is requested don't return something else.
-        @requested_responses << @available_responses.keys.first if @requested_responses.size != 1
+        # NOTE: If */* is requested - make all available, but if only one response is requested don't return something else
+        if @requested_responses.includes?("*/*")
```

Comment:
Both of these cases should just be `@available_responses.keys.first` since it will be matching against it's self.

Reply:
Agreed, given the description above.

Just to confirm, this means if someone is rolling their own headers and they request `application/json;application/xml;text/plain;` and the server does not support any of those, the response will be `text/html`. And to increase performance on 100% of requests, we don't care about this unlikely case, correct?

Reply:
I don't care about it since someone shouldn't be requesting json from an endpoint that only has html. In the past when the first 2 PR's were open on this it was discussed before.

Reply:
I'm not disagreeing or suggesting a change, just making sure I have an understanding.